src/sentry/search/eap/spans/formulas.py

```diff
@@ -377,9 +377,6 @@ def performance_score(args: ResolvedArguments, settings: ResolverSettings) -> Co
     if ratio_attribute.name == "score.total":
         return total_performance_score(args, settings)
 
-    web_vital = score_attribute.name.split(".")[-1]
-    vital_count = get_count_of_vital(web_vital, settings)
```

Comment:
Will this break individual performance scores?

I believe when you just query for the field "performance_score(lcp)", it will call this function

Reply:
riiight, makes sense!